docs/pages/usage/result.md

```diff
@@ -8,7 +8,7 @@ pagination_label: Service result
 
 # Result
 
-All services have the result of their work. For example, in case of success this call:
+All services have the result of their work. For instance, in case of success this call:
```

Comment:
It's better to use the word "example" here.

Reply:
Oh yes, thank you very much, coach